test/Microsoft.Azure.SignalR.AspNet.Tests/E2ETest/AspNetSignalRServiceE2EFacts.cs

```diff
@@ -6,11 +6,11 @@
 
 namespace Microsoft.Azure.SignalR.AspNet.Tests
 {
-    public class AspNetSignalRServiceE2EFacts : ServiceE2EFactsBase
-    {
-        public AspNetSignalRServiceE2EFacts(ITestOutputHelper output)
-            : base(new TestServer(), (string serverUrl, int count) => new TestClientSet(serverUrl, count), output)
-        {
-        }
-    }
+    //public class AspNetSignalRServiceE2EFacts : ServiceE2EFactsBase
```

Comment:
Comment is strange. Do we still need this class?

Reply:
Will uncomment it. Yes, we need this to run test in net461 framework.